src/OutputWriters/netcdf_output_writer.jl

```diff
@@ -1,135 +1,716 @@
+
+    return Dict(
+        zᵃᵃᶠ_name => zᵃᵃᶠ_attrs,
+        zᵃᵃᶜ_name => zᵃᵃᶜ_attrs,
+        Δzᵃᵃᶠ_name => Δzᵃᵃᶠ_attrs,
+        Δzᵃᵃᶜ_name => Δzᵃᵃᶜ_attrs,
+    )
+end
+
+function default_dimension_attributes(grid::RectilinearGrid, dim_name_generator)
+    xᶠᵃᵃ_name = dim_name_generator("x", grid, f, nothing, nothing, Val(:x))
+    xᶜᵃᵃ_name = dim_name_generator("x", grid, c, nothing, nothing, Val(:x))
+    yᵃᶠᵃ_name = dim_name_generator("y", grid, nothing, f, nothing, Val(:y))
+    yᵃᶜᵃ_name = dim_name_generator("y", grid, nothing, c, nothing, Val(:y))
+
+    Δxᶠᵃᵃ_name = dim_name_generator("dx", grid, f, nothing, nothing, Val(:x))
+    Δxᶜᵃᵃ_name = dim_name_generator("dx", grid, c, nothing, nothing, Val(:x))
+    Δyᵃᶠᵃ_name = dim_name_generator("dy", grid, nothing, f, nothing, Val(:y))
+    Δyᵃᶜᵃ_name = dim_name_generator("dy", grid, nothing, c, nothing, Val(:y))
+
+    xᶠᵃᵃ_attrs = Dict("long_name" => "Locations of the cell faces in the x-direction.",   "units" => "m")
+    xᶜᵃᵃ_attrs = Dict("long_name" => "Locations of the cell centers in the x-direction.", "units" => "m")
+    yᵃᶠᵃ_attrs = Dict("long_name" => "Locations of the cell faces in the y-direction.",   "units" => "m")
+    yᵃᶜᵃ_attrs = Dict("long_name" => "Locations of the cell centers in the y-direction.", "units" => "m")
+
+    Δxᶠᵃᵃ_attrs = Dict("long_name" => "Spacings between the cell centers (located at the cell faces) in the x-direction.", "units" => "m")
+    Δxᶜᵃᵃ_attrs = Dict("long_name" => "Spacings between the cell faces (located at the cell centers) in the x-direction.", "units" => "m")
+    Δyᵃᶠᵃ_attrs = Dict("long_name" => "Spacings between the cell centers (located at the cell faces) in the y-direction.", "units" => "m")
+    Δyᵃᶜᵃ_attrs = Dict("long_name" => "Spacings between the cell faces (located at the cell centers) in the y-direction.", "units" => "m")
+
+    horizontal_dimension_attributes = Dict(
+         xᶠᵃᵃ_name =>  xᶠᵃᵃ_attrs,
+         xᶜᵃᵃ_name =>  xᶜᵃᵃ_attrs,
+         yᵃᶠᵃ_name =>  yᵃᶠᵃ_attrs,
+         yᵃᶜᵃ_name =>  yᵃᶜᵃ_attrs,
+        Δxᶠᵃᵃ_name => Δxᶠᵃᵃ_attrs,
+        Δxᶜᵃᵃ_name => Δxᶜᵃᵃ_attrs,
+        Δyᵃᶠᵃ_name => Δyᵃᶠᵃ_attrs,
+        Δyᵃᶜᵃ_name => Δyᵃᶜᵃ_attrs
+    )
+
+    vertical_dimension_attributes = default_vertical_dimension_attributes(grid.z, dim_name_generator)
+
+    return merge(
+        base_dimension_attributes,
+        horizontal_dimension_attributes,
+        vertical_dimension_attributes
+    )
+end
+
+function default_dimension_attributes(grid::LatitudeLongitudeGrid, dim_name_generator)
+    λᶠᵃᵃ_name = dim_name_generator("longitude", grid, f, nothing, nothing, Val(:x))
+    λᶜᵃᵃ_name = dim_name_generator("longitude", grid, c, nothing, nothing, Val(:x))
+
+    λᶠᵃᵃ_attrs = Dict("long_name" => "Locations of the cell faces in the zonal direction.",   "units" => "degrees east")
+    λᶜᵃᵃ_attrs = Dict("long_name" => "Locations of the cell centers in the zonal direction.", "units" => "degrees east")
```

Comment:
Same comment here for the names and units.

Reply:
Ah, I now understand that these are just the default ones! Somewhere down there must be an option to provide user-defined ones. Got it! Disregard the other comments about this,